addons/website_slides/tests/test_slide_utils.py

```diff
@@ -128,3 +129,17 @@ def test_youtube_urls(self):
                     document = self.env['slide.slide']._find_document_data_from_url(url)
                     self.assertEqual(document[0], 'youtube')
                     self.assertEqual(document[1], id)
+
+@odoo.tests.tagged('archive_course')
```

Comment:
What is the use of this tag?

Reply:
It's for the --test-tags option of odoo-bin. It only start the tagged test.

Reply:
Ok so it was for your local testing.
Should be removed from the PR then :)

Reply:
Well, there is tagged test inside a lot of tests files so, from my perspective, it was ok to keep it.

Reply:
The existing tags ("post_install", "at_install") have a specific impact on the runbot (when to run the test? is the test only for external builds? ...).

You can use your custom tags to ease your local testing but they obviously should not appear in the code base.

Reply:
Tag removed :)